recipe/meta.yaml

```diff
@@ -48,9 +48,17 @@ source:
     url: https://github.com/opencv/opencv_extra/raw/master/testdata/highgui/video/VID00003-20100701-2204.avi
     sha256: 78884f64b564a3b06dc6ee731ed33b60c6d8cd864cea07f21d94ba0f90c7b310
 
+{% set build = 10 %}
+
+# Prioritize qt5 for noq over none and over qt6
+
+{% if qt_version == "5" %}
+{% set build = build + 400 %}
```

Comment:
Should we use `track_features` instead?

Reply:
The mamba way? that wasn't too popular in pytorch / tensorflow (i forget which). this is just "simpler" and easier to understand.

https://github.com/conda-forge/pytorch-cpu-feedstock/issues/71

Reply:
I remembered that @xhochy recently recommended using it: https://github.com/conda-forge/staged-recipes/pull/21247